GNUmakefile

```
@@ -97,40 +97,51 @@ telemetry-protobuf: $(telemetry-protobuf) ## Generate telemetry protocol buffers
 test_install_ginkgo: ## Install ginkgo executable to run tests
 	go install github.com/onsi/ginkgo/v2/ginkgo@v$(shell cat ./.tool-versions | grep ginkgo | sed -En "s/ginkgo.(.*)/\1/p")
 
+.PHONY: test_need_operator_assets
+test_need_operator_assets: ## Add blank file in web assets if operator ui has not been built
+	[ -f "./core/web/assets/index.html" ] || mkdir ./core/web/assets && touch ./core/web/assets/index.html
+
 .PHONY: test_smoke
 test_smoke: ## Run all integration smoke tests, using only simulated networks, default behavior
+	$(MAKE) test_need_operator_assets
```

I think you can add on any other make commands that have a PHONY by just putting them after the colon like so
```suggestion
test_smoke: test_need_operator_assets ## Run all integration smoke tests, using only simulated networks, default behavior
```